lighthouse-core/report/html/renderer/element-screenshot-renderer.js

```diff
@@ -139,24 +139,34 @@ class ElementScreenshotRenderer {
    * @param {LH.Artifacts.FullPageScreenshot} fullPageScreenshot
    */
   static installOverlayFeature(dom, templateContext, fullPageScreenshot) {
-    const reportEl = dom.find('.lh-report', dom.document());
-    const screenshotOverlayClass = 'lh-feature-screenshot-overlay';
-    if (reportEl.classList.contains(screenshotOverlayClass)) return;
-    reportEl.classList.add(screenshotOverlayClass);
-
-    const maxLightboxSize = {
-      width: dom.document().documentElement.clientWidth,
-      height: dom.document().documentElement.clientHeight * 0.75,
-    };
+    const rootEl = dom.find('.lh-root', dom.document());
+    if (!rootEl) {
+      return console.warn('No lh-root. Overlay install failed.'); // eslint-disable-line no-console
```

Comment:
nit: these `return <void>` calls give me the heebee-jeebies. reminds me of the days of express apps and callback hell. consider `return` on a separate line?

it is also a pattern that I don't think we do anywhere else.